internal/venice-common/src/main/java/com/linkedin/venice/zk/VeniceZkPaths.java

```diff
@@ -13,4 +19,8 @@ public class VeniceZkPaths {
   public static final String STORES = "Stores";
   public static final String STORE_CONFIGS = "storeConfigs";
   public static final String STORE_GRAVEYARD = "StoreGraveyard";
+  private static final Set<String> CLUSTER_ZK_PATHS_MODIFIABLE = new HashSet<>(
+      Arrays.asList(ADMIN_TOPIC_METADATA, EXECUTION_IDS, PARENT_OFFLINE_PUSHES, ROUTERS, STORE_GRAVEYARD, STORES));
+  /** Set of all Venice-managed ZooKeeper cluster paths */
```

Comment:
Can we move this comment before `CLUSTER_ZK_PATHS_MODIFIABLE` and add a blank like before the comment to signify that this is different from the variables listed above